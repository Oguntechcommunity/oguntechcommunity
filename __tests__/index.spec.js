import * as React from "react";
import { mount } from "enzyme";
import Index from "../pages/index.js";

describe("Pages", () => {
	describe("Index", () => {
		it("should match snapshot", () => {
			const wrap = mount(<Index />);
			expect(wrap).toMatchSnapshot();
		});

		it("should render without throwing an error", () => {
			const wrap = mount(<Index />);
			expect(wrap.find("h1").text()).toBe("Welcome to Next.js!");
		});
	});
});
