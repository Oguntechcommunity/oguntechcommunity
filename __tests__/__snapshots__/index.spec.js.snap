// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages Index should match snapshot 1`] = `ReactWrapper {}`;
